### PR TITLE
Updated the build system a bit

### PR DESCRIPTION
**Pull request by pawelprazak**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -133,3 +133,5 @@ lazy val uiTests = (project in file("ui-tests"))
   ) dependsOn backend
 
 RenameProject.settings
+
+fork in run := true
```
Comment (adamw) [inline build.sbt]: thanks for the PR! I'm just curious, why is the fork needed here?

File: build.sbt [hunk 2]
```diff
@@ -12,8 +12,8 @@ val logBackVersion = "1.1.6"
 val scalaLoggingVersion = "3.1.0"
 val slickVersion = "3.1.1"
 val seleniumVersion = "2.48.2"
-val circeVersion = "0.4.0-RC1"
-val akkaVersion = "2.4.2"
+val circeVersion = "0.4.0-RC2"
```
Comment (kciesielski) [inline build.sbt]: Looks like `0.4.0` is now available, can you bump this too?

File: build.sbt (hunk 1 above)
Comment (pawelprazak) [inline build.sbt]: Actually it's added itself when I've run the` activator` and after consulting [the docs](http://www.scala-sbt.org/0.13/docs/Running-Project-Code.html) I figured it couldn't hurt ;)
But after a closer look I see now it [can](https://stackoverflow.com/questions/31817729/activator-run-doesnt-work) [cause](https://github.com/typesafehub/activator/issues/1036) [problems](https://github.com/typesafehub/activator/issues/1060)

File: build.sbt (hunk 2 above)
Comment (pawelprazak) [inline build.sbt]: Sure thing.

**Comment by adamw** [comment]
Thanks!

File: build.sbt (hunk 1 above)
Comment (adamw) [inline build.sbt]: Weird that it could cause problems, but anyway if it's not necessary, no need to add it ;)

**Pull request merged**